src/main/kotlin/org/jitsi/nlj/rtp/ParsedVideoPacket.kt

```diff
@@ -29,7 +29,6 @@ abstract class ParsedVideoPacket(
 ) : VideoRtpPacket(buffer, offset, length, encodingIndex) {
 
     abstract val isKeyframe: Boolean
-    abstract val height: Int
```

Comment:
Why did this move down?

Reply:
It wasn't used for anything at the ParsedVideoPacket level

Reply:
I see.  Logically it does feel like it would belong at the `ParsedVideoPacket` level, though.

Reply:
The issue was also that it would be fairly complex to set it for a scalable stream (you have to remember and parse the scalability structure, and then correlate that with the spatial layer id) -- which is a lot of work for a field which isn't used anywhere.